File: week_02/src/main/java/Question08.java
//Autor : Zaid Nayeb
//Date  : 1404/08/26

//Question:
//Determine if a number is positive, negative, or zero Description: Use if-else statements to categorize a number.
//Example:
//Input:
//-3
//Output:
//Negative

public class Question08 {

    public static String checkNumber(double number){
        if (number > 0) {
            return "Positive";
        }else if (number < 0) {
            return "Negative";
        } else {
            return "Zero";
        }
    }
}

File: week_03/src/Question_08.js
// Author : Zaid Nayeb
// Date   : 2025/11/26
/*
8.	Write a function to check if an array contains a specific element Description: Determine if a given element exists in the array. 
Example: 
Input: 
Array = [3, 6, 9, 12], Element = 6 Output: 
Found 

*/

function containsElement(arr, element) {
  for (let i = 0; i < arr.length; i++) {
    if (arr[i] === element) {
      return "Found";
    }
  }
  return "Not Found";
}

console.log(containsElement([3, 6, 9, 12], 6)); 
console.log(containsElement([3, 6, 9, 12], 5)); 


module.exports = containsElement;